### DISCUSSION TITLE
Goal 5: Achieve gender equality and empower all women and girls

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#27] [General]
Chat about anything related to Goal 5: [Achieve gender equality and empower all women and girls](https://github.com/Project-Resilience/platform/blob/main/goals/goal_05.md)

**Comment by ofrancon** [comment]
See this discussion in Goal 10: https://github.com/Project-Resilience/platform/discussions/32#discussioncomment-15828390

**Reply by ofrancon** [reply]
And the corresponding project proposal: [Reducing Investor Gender Bias in Startup Funding](https://github.com/Project-Resilience/platform/blob/main/proposals/investor_gender_bias.md)

**Reply by ofrancon** [reply]
Related report: [Inclusive Alpha Founder Report](https://www.adaventures.com/inclusive-alpha-founder-report)
